browser/hooks/useSolanaToEvmSwap.ts

```diff
@@ -228,8 +231,31 @@ export const useSolanaToEvmSwap = (
       }
 
       /**
-       * STEP 7: Subscribe to events on target chain
+       * STEP 7: Subscribe to events on source and target chain
        */
+      const subscriptionId = solanaConnection.onLogs(
```

Comment:
Shouldn't need this in this direction because we just wait to confirm/fetch the tx.